modules/@amperka/usb-keyboard.js

```diff
@@ -236,13 +233,128 @@ var type = function(txt, callback) {
       } else {
         tap('?');
       }
-
       txt = txt.substr(1);
     }
-  }, 20);
+  }, timeForWait);
 }
 
+var pressedKeys = [0, 0, 0, 0, 0, 0, 0, 0];
+
+var chooseKey = function(keyChanger, opts) {
+  let chrCode;
+  if (opts in MODIFY) {
+    keyChanger([MODIFY[opts], 0]);
+  } else {
+    chrCode = opts.charCodeAt(0);
+    if (chrCode >= 97 && chrCode <= 122) {
+      // lower case letter a-z
+      keyChanger(opts.toUpperCase());
+    } else if (chrCode >= 65 && chrCode <= 90) {
+      // upper case letter A-Z
+      keyChanger(opts);
+    } else if (opts in KEY) {
+      keyChanger(KEY[opts]);
+    } else if (opts in SHIFT_KEYS) {
+      keyChanger(SHIFT_KEYS[opts]);
+    } else {
+      keyChanger('?');
+    }
+  }
+}
+
+var press = function(opts, callback) {
+  chooseKey(changePressedKeys, opts);
+  if (callback) {
+    callback();
```

Comment:
See no reason for `callback` in its current form since it is called in :100: of cases, and always synchronously.